Jamulus.pro

```diff
@@ -95,20 +95,39 @@ win32 {
     contains(CONFIG, "jackonwindows") {
         message(Using Jack instead of ASIO.)
 
-        !exists("C:/Program Files (x86)/Jack/includes/jack/jack.h") {
-            message(Warning: jack.h was not found at the usual place, maybe jack is not installed)
+        contains(QT_ARCH, "x86") {
```

Comment:
If I install Windows 32bit, isn't "Program Files" where 32bit programs live?

I thought "Program Files (x86)" only existed on Windows 64bit systems (where "Program Files" is for 64bit programs and "Programs Files (x86)" for 32bit).

Reply:
I don't know. If anyone can say something about that. 
(jujudusud is thinking of Microsoft just deprecated 32 bits windows 10...)
Anyway, If this is wrong I will change this line. Thanks for the review !

Reply:
> If I install Windows 32bit, isn't "Program Files" where 32bit programs live?
> 
> I thought "Program Files (x86)" only existed on Windows 64bit systems (where "Program Files" is for 64bit programs and "Programs Files (x86)" for 32bit).

That is correct, as long as you compile/build on the solution on a 64-bit system then this should not be an issue.

Reply:
If you're building Jamulus x64, then we want JACK x64 (which will be under "Program Files" on x64).  QT_ARCH won't be "x86".

The main reason anyone would be building x86 would be if they were building on x86 and want JACK x86 (which will be under "Program Files" on x86).  QT_ARCH will be "x86".

The only time to use "Program Files (x86)", then, is if cross-compiling from x64 to x86.  However, QT_ARCH alone doesn't tell you this, from what I can see?

Reply:
So...

If QT_ARCH = x86
* If  "Program Files (x86)" exists, it's a cross-compile - set the install dir to "Program Files (x86)"
* Otherwise set the install dir to "Program Files"
* Set the x86-specific bits

Else
* Set the install dir to "Program Files"
* Set the x64-specific bits

Check the install is sane.

Reply:
@pljones when you build the instapper for Windows, it includes both the x86 and x86_x64 application in it. When installing this the appropiate version for your pc will be installen. So you need to be able to have both options at the same time.

Reply:
I don't how to code this into the Jamulus.pro file.

Reply:
Ok with :
c0a5110